modules/ocf_www/files/matrix-client

```diff
@@ -0,0 +1,9 @@
+{
+    "m.homeserver": {
+        "base_url": "https://matrix.ocf.berkeley.edu",
+        "server_name": "ocf.berkeley.edu"
+    },
+    "m.identity_server": {
+        "base_url": "https://vector.im"
```

Comment:
Doesn't identity come from keycloak? (perhaps this is a matrix thing I don't know about)

Reply:
From my understanding, the identity servers in Matrix are so other users can discover you through your phone or email (if connected). I just copied it from the [existing config we use for the hosted Element instance](https://github.com/ocf/matrix/blob/d9db94473f8daa517208e5c6ec1aa9189f9fe4fe/config.riot.json), but I can remove it if you'd like since this portion is optional. Most clients already default to vector.im anyways.

Reply:
Aha, I see, that makes more sense.